### PR TITLE
Use ffz/bttv channel emotes for chatrooms

### PR DESCRIPTION
**Pull request by hemirt**
chatrooms being identified by their twitch format of `chatrooms:<channel` ID>:<room UUID>`

### CODE REVIEW

File: src/providers/twitch/ChatroomChannel.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#include "ChatroomChannel.hpp"
+
+#include <QDebug>
+#include "TwitchApi.hpp"
+#include "common/Common.hpp"
+#include "messages\Emote.hpp"
```
Comment (pajlada) [inline src/providers/twitch/ChatroomChannel.cpp]: this backslash in the include looks off, change it to a normal forward slash

File: src/providers/twitch/ChatroomChannel.cpp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace chatterino {
+
+ChatroomChannel::ChatroomChannel(const QString &channelName,
+                                 TwitchBadges &globalTwitchBadges,
+                                 BttvEmotes &globalBttv, FfzEmotes &globalFfz)
+    : TwitchChannel(channelName, globalTwitchBadges, globalBttv, globalFfz)
+{
+    auto list = channelName.split(":");
+    if (list.size() > 2)
```
Comment (pajlada) [inline src/providers/twitch/ChatroomChannel.cpp]: what would be the consequence of a chatroom id such as `chatrooms:123456` being input? the first condition at https://github.com/fourtf/chatterino2/pull/966/files#diff-6f295e4ab67461a536026941daa7aab5R97 would be fulfilled, but then this one wouldn't. would it be smarter to have a freestanding function which ensures the validity of the chatroom syntax before attempting to create a chatroom channel?

File: src/providers/twitch/ChatroomChannel.cpp (hunk 1 above)
Comment (hemirt) [reply src/providers/twitch/ChatroomChannel.cpp]: fixed

File: src/providers/twitch/ChatroomChannel.cpp (hunk 2 above)
Comment (hemirt) [reply src/providers/twitch/ChatroomChannel.cpp]: any particular place where to put the  [isChatroom](https://github.com/fourtf/chatterino2/blob/3c01af980579db1e97c9393446a257a412cf70af/src/providers/twitch/TwitchServer.cpp#L26-L39) function?

**Comment by hemirt** [comment]
fixed 
> {time} chatrooms:{channel_id}:{room_id} is live
message to show the channel owner name

File: src/providers/twitch/ChatroomChannel.cpp (hunk 2 above)
Comment (pajlada) [reply src/providers/twitch/ChatroomChannel.cpp]: I think the current positioning is fine :+1:

**Comment by pajlada** [comment]
Works great :+1: Thank you!

**Pull request merged**